lib/matplotlib/backend_bases.py

```diff
@@ -1510,6 +1510,17 @@ def _draw(renderer): raise Done(renderer)
             return figure._cachedRenderer
 
 
```

Comment:
Would it make more sense to add this functionality as a parameter to `_get_renderer`, i.e. something like `_get_renderer(..., disabled_draw=True)`?

Reply:
sure, sounds good, done.